client/src/app/App.js

```diff
@@ -1670,6 +1670,10 @@ export class App extends PureComponent {
       return this.createDiagram('cloud-bpmn');
     }
 
+    if (action === 'create-cloud-dmn-diagram') {
```

Comment:
Should we add [a test here](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/__tests__/AppSpec.js#L281)?

Reply:
👍

Reply:
It's already there: https://github.com/camunda/camunda-modeler/pull/2845/files#diff-b86fdef65a943262634366fd9d7fe4c16c071abbda264ab9aae9f1bc7375c2caR306

😅

Reply:
Oh, codecov fooled me then 😄